### PR TITLE
Bugfixes and overall experience improvements

### PR DESCRIPTION
**Pull request by nteyan**
## 🛠️ Description

- Queries performance optimization (Cost Management tab)
- Improved experience to manage costs filtered by tag (Cost Management tab)

Fixes #(issue)

- Fixes applied on Tag Explorer queries
- Subscription filter updated on Compliance tab to restrict the result to one subscription only

## 🔬 How has this been tested?

- [ ] 🫰 PS -WhatIf / az validate
- [X] 👍 Manually deployed + verified
- [ ] 💪 Unit tests

### CODE REVIEW

File: src/workbooks/governance/workbook.json [hunk 1]
```diff
@@ -2122,21 +2240,24 @@
                         },
                         "queryType": 1,
                         "resourceType": "microsoft.resourcegraph/resources",
-                        "value": null
+                        "value": "TOTO"
```
Comment (flanakin) [inline src/workbooks/governance/workbook.json:2243]: TODO?
```suggestion
                        "value": "TODO"
```

File: src/workbooks/governance/workbook.json [hunk 2]
```diff
@@ -2453,15 +2597,21 @@
...
+                            "value::all"
+                          ],
                           "selectAllValue": "*",
                           "showDefault": false
                         },
                         "queryType": 0,
                         "resourceType": "microsoft.operationalinsights/workspaces",
-                        "value": null
+                        "value": [
+                          "/subscriptions/3D2171D3-2B1E-41D6-A62C-DC780B0C5A04"
```
Comment (flanakin) [inline src/workbooks/governance/workbook.json:2613]: Another sub ID...

File: src/workbooks/governance/workbook.json [hunk 3]
```diff
@@ -4614,17 +4848,80 @@
...
+                    "version": "KqlParameterItem/1.0",
+                    "crossComponentResources": [
+                      "value::tenant"
+                    ],
+                    "parameters": [
+                      {
+                        "id": "f967854d-c31e-46a6-b8b0-6b9fff7ce582",
+                        "version": "KqlParameterItem/1.0",
+                        "name": "ManagementGroup",
+                        "label": "Management Group",
```
Comment (flanakin) [inline src/workbooks/governance/workbook.json:4863]:
```suggestion
                        "label": "Management group",
```

File: src/workbooks/governance/workbook.json [hunk 4]
```diff
@@ -5009,7 +5316,7 @@
                         "timeContext": {
                           "durationMs": 86400000
                         },
-                        "value": "/subscriptions/e98fff73-a41f-4746-a887-e0423ff6c2f4"
+                        "value": "/subscriptions/3d2171d3-2b1e-41d6-a62c-dc780b0c5a04"
```
Comment (flanakin) [inline src/workbooks/governance/workbook.json:5319]: Sub ID...

**Review by flanakin — APPROVED** [review]
All of my comments are nits. Feel free to handle them as you see fit.

File: src/workbooks/governance/workbook.json [hunk 5]
```diff
@@ -156,17 +160,23 @@
...
                     "showDefault": false
                   },
                   "timeContext": {
                     "durationMs": 86400000
                   },
                   "queryType": 1,
                   "resourceType": "microsoft.resources/tenants",
-                  "value": ["value::all"]
+                  "value": [
+                    "/subscriptions/3d2171d3-2b1e-41d6-a62c-dc780b0c5a04"
```
[the remark this answers is not in the document]
Comment (nteyan) [reply src/workbooks/governance/workbook.json:178]: I'm going to commit a new version with all sub ID cleaned. I will take care of it for the next commits

File: src/workbooks/governance/workbook.json (hunk 2 above)
Comment (nteyan) [reply src/workbooks/governance/workbook.json:2613]: cleaned also in my next commit

File: src/workbooks/governance/workbook.json (hunk 4 above)
Comment (nteyan) [reply src/workbooks/governance/workbook.json:5319]: cleaned

**Pull request merged**